app/views/markets/_index.html.erb

```diff
@@ -0,0 +1,7 @@
+<h1> MARKETS!! </h1>
+
+
+  <% @markets = Market.all %>
```

Comment:
You already define @markets in your controller, so you don't need to do this here. Best practice is to gather and process all your data in the model and controller and keep this logic out of the view.